Sources/RxAlamofire.swift

```diff
@@ -775,11 +775,11 @@ extension Reactive where Base: DataRequest {
                 case .success(let result):
                     if let httpResponse = packedResponse.response {
                         observer.on(.next(httpResponse, result))
+                        observer.on(.completed)
                     }
                     else {
                         observer.on(.error(RxAlamofireUnknownError))
                     }
-                    observer.on(.completed)
```

Comment:
@freak4pc 

> line 778 emits completion immediately after next

that's true but if you check code before patch, you'll see that it got to line 778 from line 782, which was executed immediately after next before, i.e. my statement still holds - it doesn't change anything regarding streaming support